docs/UserGuide.md

```diff
@@ -255,5 +282,6 @@ Action | Format, Examples
 **Edit Room** | `editroom ROOM_NUMBER [r/NEW_ROOM_NUMBER] [p/PATIENT_NAME]` <br> e.g., `editroom 1 r/2 p/alex`
 **Add Task to Room** | `addtask d/DESCRIPTION r/ROOM_NUMBER [dd/DUE_DATE]` <br>
 **Delete Task from Room** | `deletetask r/ROOM_NUMBER t/TASK_NUMBER` <br>
+**Edit Task in Room** | `edittask r/ROOM_NUMBER t/TASK_NUMBER [d/DESCRIPTION] [dd/DUE_DATE]` <br>
 **Search Task** | `searchtask dd/DUE_DATE` <br>
 **Help** | `help`
```

Comment:
We rearrange later in alphabetical order

Reply:
Hmm... Should it be in alphabetical order? I was thinking maybe it should be sorted by the type. We can have `Patient`, followed by `Room`, then `Task`.

```
3.2 Patient
      3.2.1 Add Patient
      3.2.2 Delete Patient
      ...
3.3 Room
      3.3.1 Initialise Room
      ...
3.4 Task
      3.4.1 Add Task
      3.4.2 Delete Task
```

Reply:
i think we will have to split them into subcategory at a later date, maybe we can list the feature in the order of how the user might user them? all these can be discussed at a later date i think